spec/factories/ext_management_system.rb

```diff
@@ -209,7 +208,11 @@
   factory :configuration_manager_ansible_tower,
           :aliases => ["manageiq/providers/ansible_tower/configuration_manager"],
           :class   => "ManageIQ::Providers::AnsibleTower::ConfigurationManager",
-          :parent  => :configuration_manager
+          :parent  => :configuration_manager do
+            after :create do |x|
```

Comment:
@syncrou I think this should be in a trait so that we don't always create ConfigurationScripts